specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyEvents.json

```diff
@@ -0,0 +1,970 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyEventsClient",
+    "version": "2019-10-01"
```

Comment:
>2019-10-01 [](start = 16, length = 10)

I think it is not yet implemented. I suggest move this file under 2018-07-01-preview. And add here once policyEvents is implemented for this version.

Reply:
rather better, we could skip adding it to preview and directly move to latest version for policyEvents. 

---
In reply to: [355050603](https://github.com/Azure/azure-rest-api-specs/pull/7909#discussion_r355050603) [](ancestors = 355050603)

Reply:
had added to 2019-10 as you had suggested, will move it to 2018-07-01-preview since its not implemented yet.